code/src/components/HappyForm.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+
+const HappyForm = ({ handleFormSubmit, newThought, handleNewThoughtChange }) => {
+  const isSubmitButtonDisabled = newThought.length < 6 || newThought.length > 140
+
+  return (
+    <form className="form" onSubmit={handleFormSubmit}>
+      <label htmlFor="thought-input">What is making you happy right now?</label>
+      <textarea
+        className="thought-input"
```

Comment:
Here you could use this snipet to introduce a color change in text area if the user type more than 140:

`style={newThought.length > 140 ? { color: 'red' } : {}}`